### DISCUSSION TITLE
A Libfabric Slack workspace

### DISCUSSION THREAD

**Discussion by sunkuamzn** [ofiwg/libfabric#9114] [Ideas]
Open MPI and MPICH have Slack workspaces which are quite handy for discussions and getting help. Would be great if Libfabric had one too!

**Comment by qkoziol** [comment]
I'd like to get this priority bumped also.  AWS can even create it and invite everyone.

**Comment by shefty** [comment]
No objection here.

**Comment by qkoziol** [comment]
Done!  At:  libfabric.slack.com
